_posts/2021-09-26-boa-release-13.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: "Boa release v0.13"
+author: Boa Developers
+---
+
+Boa v0.13 is here! Boa is a JavaScript parser, compiler and executor written in the Rust programming language. It makes it easy to embed a JS engine in your projects, and you can even use it from webassembly. See the [About](/about) page for more info.
+
+We currently support part of the language. In this release, our conformance has grown to 41.97% of the official ECMAScript Test Suite (Test262). In this release, we have closed ?? issues and merged ?? pull requests. You can check the full list of changes [here](https://github.com/boa-dev/boa/blob/v0.13/CHANGELOG.md).
+
+## New Features
+
+This release brings some new features, such as support for the [Relative Indexing Method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at) (better known as `at()`). Thanks to @neeldug for adding this. Other additions include `replaceAll()`, `Object.values()`, `Object.preventExtensions()`, `splice()`, `sort()`, spreading objects and more.
+
+## Test 262
+
+Test262 is the implementation conformance test suite maintained by TC39. It's used by nearly all engines to measure how conformant they are to the specification. Boa pulls the tests in-tree and runs them against all PRs. You can find more about Test262 [here](https://github.com/tc39/test262).
+
+Since v0.11 we have 10% more conformance and reduced panics. To put that into perspective that's almost 10,000 more tests that pass since when version 0.11 was released. We've dropped from 26 tests panicing to zero.
```

Comment:
I guess here we should compare it with 0.12

Reply:
Yeah I do you know if we have comparisons vs 0.12?

Reply:
I looked it up, the last commit for 0.12 was https://github.com/boa-dev/boa/commit/33a69654d07cf1d935cf4f48cc742579f9321d88. Here are the numbers:

|   | 0.12 | 0.13 | diff |
|---|---|---|---|
| Total | 78897 | 80918 | +2021 |
| Passed | 26801 | 33192 | +6391 |
| Ignored | 15628 | 15898 | +270  |
| Panics |0 | 0 | 0 |
| Failed | 36468 | 31828 | -4640 |
| Conformance | 33,97% | 41,02% | +7,05% |